packages/core/src/components/forms/_form-group.scss

```diff
@@ -0,0 +1,118 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Form group
+
+Add additional helper text to help users navigate forms and complex fields.
+
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+  Note that this feature necessitates additional DOM elements,
+  thus requiring a more complex form structure achieved with `.pt-form-group`.
+</div>
+
+- Link each label to its respective control element with a `for={#id}` attribute on the `<label>` and
+`id={#id}` on the control.
+
+- Add extra information to the label with `span.pt-text-muted`.
```

Comment:
actually this should be possible with simple labels too so i've moved it up to that section (and needs a class to help with the disabled state)